Makefile

```diff
@@ -0,0 +1,121 @@
+# Requirements
+# docker or podman installed and running
+# git installed
+# .git_token file exists with current Token (or injected as env var)
+# Linux OS
+
+ifndef WORKFLOW_ID
+ifneq ($(MAKECMDGOALS),help)
+$(error WORKFLOW_ID variable is not defined. Please provide the required value.)
+endif
+endif
+
+CONTAINER_ENGINE ?= $(shell which podman >/dev/null 2>&1 && echo podman || echo docker)
+ifneq (,$(wildcard $(CURDIR)/.docker))
+	DOCKER_CONF := $(CURDIR)/.docker
+else
+	DOCKER_CONF := $(HOME)/.docker
+endif
+
+WORKDIR ?= $(shell mktemp -d)
+ifeq ($(shell uname),Darwin)
+	# Use a fixed folder to simplify limactl configuration (must be mounted with Write permissions)
+	WORKDIR := /tmp/serverless-workflows
+endif
+SCRIPTS_DIR := scripts
+
+GIT_USER_NAME ?= $(shell git config --get user.name)
+GIT_USER_EMAIL ?= $(shell git config --get user.email)
+GIT_REMOTE_URL := $(shell git config --get remote.origin.url)
+GIT_REMOTE_URL := $(shell echo "$(GIT_REMOTE_URL)" | sed 's/\.git//')
+GIT_REMOTE_URL := $(shell echo "$(GIT_REMOTE_URL)" | sed 's/git@/https:\/\//')
+GIT_REMOTE_URL := $(shell echo "$(GIT_REMOTE_URL)" | sed 's/com:/com\//')
+PR_OR_COMMIT_URL ?= "$(GIT_REMOTE_URL)/commits/$(shell git rev-parse --short=8 HEAD)"
+GIT_TOKEN ?= $(shell cat .git_token)
+
+LINUX_IMAGE ?= quay.io/orchestrator/ubi9-pipeline:latest
+
+WORKFLOW_ID := ""
+IMAGE_NAME := ""
+REGISTRY ?= quay.io
+REGISTRY_REPO ?= $(shell id -un)
+REGISTRY_USERNAME ?= ""
+REGISTRY_PASSWORD ?= ""
+IMAGE_PREFIX ?= serverless-workflow
+IMAGE_TAG ?= $(shell git rev-parse --short=8 HEAD)
+DOCKERFILE = pipeline/workflow-builder.Dockerfile
+
+IMAGE_NAME = $(REGISTRY)/$(REGISTRY_REPO)/$(IMAGE_PREFIX)-$(WORKFLOW_ID)
+
+# TODO use parodos-dev
+DEPLOYMENT_REPO ?= dmartinol/serverless-workflows-helm
+
+.PHONY: help
+.PHONY: all
+
+all: push-image push-manifests
+
+# Target: prepare-workdir
+# Description: copies the local repo content in a temporary WORKDIR for file manipulation.
+# Usage: make prepare-workdir
+prepare-workdir:
+	@echo "Preparing workdir $(WORKDIR)"
+	@rm -rf $(WORKDIR)
+	@mkdir -p $(WORKDIR)
+	@cp -R . $(WORKDIR)
+
+# Target: build-image
+# Description: Builds the workflow containerized image from the given WORKDIR.
+# Depend on: prepare-workdir target.
+# Usage: make build-image
+build-image: BUILD_ARGS=--build-arg WF_RESOURCES=$(WORKFLOW_ID) \
+	--build-arg QUARKUS_EXTENSIONS=org.kie.kogito:kogito-addons-quarkus-jobs-knative-eventing:999-SNAPSHOT
```

Comment:
isn't `org.kie.kogito:kogito-addons-quarkus-jobs-knative-eventing:999-SNAPSHOT` included in the builder image? or at least expected to be added?